### PR TITLE
fix v4: 'scaling' event triggered before object position is adjusted

### PR DESCRIPTION
**Pull request by SLKnutson**
Hopefully this is satisfactory. It worked in my project.
https://github.com/fabricjs/fabric.js/issues/6649
Thanks!

### CODE REVIEW

File: src/controls.actions.js [hunk 1]
```diff
@@ -188,15 +188,19 @@
   /**
    * Wrap an action handler with saving/restoring object position on the transform.
    * this is the code that permits to objects to keep their position while transforming.
+   * @param {string} eventName
    * @param {Function} actionHandler the function to wrap
    * @return {Function} a function with an action handler signature
    */
-  function wrapWithFixedAnchor(actionHandler) {
+  function wrapWithFixedAnchor(eventName, actionHandler) {
```
Comment (asturur) [inline src/controls.actions.js:195]: we can't change wrapWithFixedAnchor argument signature, or we would break the compatibility of fabric v4.

Comment (asturur) [reply src/controls.actions.js:195]: we need an event wrapper, or something like that. I ll get some fixes done and then i ll try to help.

**Comment by asturur** [comment]
This still a valid issue

**Comment by asturur** [comment]
I do not have the pipeline to check it, or the time to fix the test errors, but we definitely need this as a reminder to fix it.

**Comment by SLKnutson** [comment]
@asturur I updated the PR to not change the argument signature

**Comment by asturur** [comment]
hi @SLKnutson i know a lot of time passed, but now the pipeline is working again.
If you can update your PR with a fresh master branch, it would run again and we can continue to fix it.

**Comment by SLKnutson** [comment]
@asturur 
It seems like the failure isn't related to the code?

**Comment by asturur** [comment]
hi @SLKnutson i ll review asap

**Comment by asturur** [comment]
seems great tho

**Comment by ShaMan123** [comment]
+1 exactly what I need. Bleeded a day on this.

**Comment by melchiar** [comment]
Thanks for this! Turns out this was the cause of a bug in my application that started a few versions ago.

**Pull request merged**